examples/nextjs-with-typescript/src/app/media-store-hooks/ReactPlayer.tsx

```diff
@@ -0,0 +1,258 @@
+'use client';
+import type { ReactNode } from 'react';
+import { useState } from 'react';
+/* Uncomment this to log out the latest mediaState as it changes */
+// import { useEffect, useState } from 'react';
+import {
+  Provider as MediaChromeProvider,
+  useMediaDispatch as useDispatch,
+  useFullscreenRefCallback,
+  useMediaRefCallback,
+  useMediaSelector as useSelector,
+} from 'media-chrome/react/media-store';
+import { constants } from 'media-chrome';
+const { MediaUIEvents } = constants;
```

Comment:
Should we consider re-exporting this from `/react/media-store` and renaming to e.g. `MediaActionTypes`?